tests/src/test/scala/org/apache/openwhisk/core/database/ArtifactWithFileStorageActivationStoreTests.scala

```diff
@@ -77,14 +77,19 @@ class ArtifactWithFileStorageActivationStoreTests()
           "2018-03-05T02:10:38.196754258Z stdout: second log line of multiple lines")))
   }
 
-  def expectedFileContent(activation: WhiskActivation) = {
+  def expectedFileContent(activation: WhiskActivation, includeResult: Boolean) = {
     val expectedLogs = activation.logs.logs.map { log =>
       JsObject(
         "type" -> "user_log".toJson,
         "message" -> log.toJson,
         "activationId" -> activation.activationId.toJson,
         "namespaceId" -> user.namespace.uuid.toJson)
     }
+    val expectedResult = if (includeResult) {
+      JsString(activation.response.result.getOrElse(JsNull).compactPrint)
+    } else {
+      s"Activation record '${activation.activationId}' for entity '${activation.name}'".toJson
```

Comment:
this is also a JsString() to avoid .toJson (only because of the line above)... nit feel free to ignore.